packages/core/content-manager/server/controllers/collection-types.js

```diff
@@ -432,7 +432,38 @@ module.exports = {
       return ctx.forbidden();
     }
 
-    const number = await entityManager.getNumberOfDraftRelations(id, model);
+    const number = await entityManager.countDraftRelations(id, model);
+
+    return {
+      data: number,
+    };
+  },
+  async countManyEntriesDraftRelations(ctx) {
+    const { userAbility } = ctx.state;
+    const ids = JSON.parse(ctx.request.query.ids);
```

Comment:
request query ids should be parsed ( I think ) , could you check?

Reply:
No I need the JSON parse because `ctx.request.query.ids` is a string